keras/engine/topology.py

```diff
@@ -1229,6 +1229,45 @@ def count_params(self):
                                    self.name + '.build(batch_input_shape)`.')
         return sum([K.count_params(p) for p in self.weights])
 
+    def count_flops(self):
```

Comment:
That sounds good. There may be confusion with the concept of `op` in TF though (e.g. a dot product). But maybe not a big deal.